AmahiAnywhere/AmahiAnywhere/SettingTableViewController.swift

```diff
@@ -0,0 +1,146 @@
+//
+//  SettingTableViewController.swift
+//  AmahiAnywhere
+//
+//  Created by Syed on 21/02/18.
+//  Copyright © 2018 Amahi. All rights reserved.
+//
+
+import UIKit
+import MessageUI
+
+class SettingTableViewController: UITableViewController ,MFMailComposeViewControllerDelegate {
+    
+    
+    //MARK:- Mail Configuration
+    
+    
+    func configureMailComposeViewController() ->MFMailComposeViewController {
+        
+        let mail = MFMailComposeViewController()
+        mail.mailComposeDelegate = self as? MFMailComposeViewControllerDelegate
+        mail.setToRecipients(["support@amohi.org"])
```

Comment:
You can add all in app texts to the StringLiterals File. Check out https://github.com/amahi/ios/blob/master/AmahiAnywhere/AmahiAnywhere/Data/Local/StringLiterals.swift